main.yml

```diff
@@ -1,5 +1,6 @@
 ---
 - hosts: base
+  gather_facts: yes
```

Comment:
I've added `gather_facts: yes` everywhere for consistency in style, although it might not be necessary for `base` related tasks

Reply:
NP, I think consistency is important, so I'll apply with your suggestion and we'll see how it goes.